documentation/library_guide/macros.rst

```diff
@@ -34,8 +34,14 @@ Use these macros to test presence of specific |onedpl_short| functionality.
 ================================== ===============================================
 Macro                              Macro values and the functionality
 ================================== ===============================================
-``ONEDPL_HAS_RANDOM_NUMBERS``      ``202210L`` for pseudo-random number generators and distributions,
-                                   as defined in |onedpl_short| version 2022.2 or later
+``ONEDPL_HAS_RANDOM_NUMBERS``      Pseudo-random number generators and distributions.
+
+                                   * ``202409L`` - added support of comparison and I/O stream operators and an experimental Philox engine
```

Comment:
Should we be listing previous values with there descriptions here as well, or just the current one?
`202210L`

Reply:
We will be listing all values, but this is the first one, as  `202210L` was never released.

Reply:
Aha whoops, you are right.  I mistook a delta commit, and thought perhaps it existed in some form prior.  
Overall LGTM.